terra_notebook_utils/drs.py

```diff
@@ -156,6 +158,32 @@ def copy(drs_url: str,
     else:
         copy_to_local(drs_url, dst, workspace_name, google_billing_project)
 
+def copy_batch(drs_urls: Iterable[str],
+               dst: str,
+               workspace_name: Optional[str]=WORKSPACE_NAME,
+               google_billing_project: Optional[str]=WORKSPACE_GOOGLE_PROJECT):
+    enable_requester_pays(workspace_name, google_billing_project)
+    with ThreadPoolExecutor(max_workers=3) as oneshot_executor:
```

Comment:
Max workers looks like a magic number here.  Should `ThreadPoolExecutor(max_workers=3)` and `AsyncSet(oneshot_executor, concurrency=3)` be set with a default variable (as with MULTIPART_THRESHOLD?).  Or an input arg with a default of 3?  And/or a function of the number of cores the machine has?

Reply:
good point. will fix.